doc/whats_new.rst

```diff
@@ -90,6 +90,10 @@ Bug fixes
      `#6497 <https://github.com/scikit-learn/scikit-learn/pull/6497>`_
      by `Sebastian Säger`_
 
+   - Fix bug svm's decision values when `decision_function_shape` is
+     `'ovr'` :class:`svm.SVC`.
+     `#7724 <https://github.com/scikit-learn/scikit-learn/pull/7724>`_
```

Comment:
@raghavrv sorry, I didn't understand this message, what do you mean by use the new :issue: tag, must i create a new issue?